features/cart/shopping_cart/removing_cart_item.feature

```diff
@@ -7,25 +7,35 @@ Feature: Removing cart item from cart
     Background:
         Given the store operates on a single channel in "United States"
         And the store has a product "T-shirt banana" priced at "$12.54"
-        And I added product "T-shirt banana" to the cart
 
-    @ui
+    @ui @api
     Scenario: Removing cart item
+        Given I added product "T-shirt banana" to the cart
```

Comment:
Why did you move this step from background as it is use in every scenario?

Reply:
Because if I use step from api context between setup context steeps I have problem with doctrine like this: 
<img width="1440" alt="Screenshot 2020-06-29 at 11 41 07" src="https://user-images.githubusercontent.com/39232096/86088442-a8598d00-baa6-11ea-9b19-b5adfda3b85b.png">